viewing_party/party.py

```diff
@@ -1,23 +1,112 @@
-# ------------- WAVE 1 --------------------
-
+# WAVE 1
 def create_movie(title, genre, rating):
-    pass
+    movie = {}
+    if not title or not genre or not rating:
+        return None
+    movie["title"] = title
+    movie["genre"] = genre
+    movie["rating"] = rating
+    return movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)

```

Comment:
[suggestion] When working with dictionary keys (which can be typo prone) it's sometimes helpful to make constants that our editor is better able to suggest for us. If we had

```python
KEY_WATCHLIST = "watchlist"
```

defined elsewhere, we could write

```python
    user_data[KEY_WATCHLIST].append(movie)
```

We might get better suggestions from our editor, reducing typos. If in the future a key ever needed to change, we'd also only need to change it in one place. Refactoring shortcuts can also help, in a good editor, but sometimes we can help things along ourselves!